src/MinterGateway.sol

```diff
@@ -559,20 +555,6 @@ contract MinterGateway is IMinterGateway, ContinuousIndexing, ERC712Extended {
         return _minterStates[minter_].updateTimestamp;
     }
 
-    /// @inheritdoc IMinterGateway
```

Comment:
why getters are gone?

Reply:
- it was inconsistent that we gave getters for missed interval penalties (`getPenaltyForMissedCollateralUpdates`), and not undercollateralized penalties
- adding the `getPenaltyForUndercollateralization` would have been tricky and needed to much nw internal code, so better to not have any
- on the call, it was suggested that off-chain (i.e. mostly minters) could determine these values for themselves if they planned on being delinquent
- these penalty getters assumed `block.timestamp`, when external users more likely cares to pass in a timestamp of their choosing
- `collateralPenaltyDeadlineOf` was weirdly named, and ambiguous in its return, and like above, did not contribute much

Anyway, we can see which of these removals, if any, we'd like to revert in this PR

Reply:
same as in the previous comment, not necessary. Should be discussed separately, not be merged with main change. It distracts attention of reviewers and auditors from the main bug fix